other/require-topologyspreadconstraints/tsc-policy.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: spread-pods
+  annotations:
+    policies.kyverno.io/title: Spread Pods Across Nodes & Zones
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/subject: Deployment, StatefulSet
+    policies.kyverno.io/description: >-
```

Comment:
Description states 3 things:
1. What is the context
2. How does this policy add value
3. What does this policy do